pkg/component/web.go

```diff
@@ -95,12 +98,14 @@ func (c *Component) initWeb() error {
 
 	if c.config.HTTP.Health.Enable {
 		g := web.RootRouter().NewRoute().Subrouter()
+		g.Use(ratelimit.HTTPMiddleware(c.RateLimiter(), "http:admin"))
```

Comment:
1. Let's just call this `health`; that's also what we call it in config.
2. Let's only rate limit when password protection is enabled